recipes-wpe/wpeframework/wpeframework-plugins_git.bb

```diff
@@ -44,6 +46,7 @@ PACKAGECONFIG ?= " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'playready_vg',         'opencdmi_vgrdm', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'widevine',             'opencdmi_wv', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'thunder',              'network', '', d)} \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'firmwarecontrol',      'firmwarecontrol', '', d)} \
```

Comment:
Same here, no need for distro features.

You can add it to package config if the default compiles / works on RPI. If it is for non-RPI (i.e. broadcom builds) then you can leave it out and it's up to the specific adaptations layers to add it into the Package Config of WPEFramework Plugins.

Reply:
it will not work by default for rpi, it work only with rdk builds which support mfr, so I will remove it from the package config and users can add it through the additional layers based on requirements